Fancy Hats/assets/css/styles.css

```diff
@@ -0,0 +1,136 @@
+/* UNIVERSAL STYLES */
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+/* BODY BACKGROUND STYLES */
+body{
+    height: 100vh;
+    width: 100vw;
+    background-image: url("/Fancy Hats/assets/Images/background-image.png") ;
+    background-size: cover;
+}
+/* WHOLE CONTAINER STYLES */
+.contentContainer{
+    display: flex;
+    justify-content: space-between;
+    padding-top: 50px;
+    height: 100vh;
+    padding-bottom: 50px;
+}
+
+/* NAVIGATION SECTION */
+.navSection{
+    position: relative;
+    width: 275px;
+}
+.hamburgerIcon{
+    display: flex;
+    padding: 32px;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+    background-color: black;
+    height: 100px;
+    width: 100px;
+    cursor: pointer;
+}
+.hamburgerIcon span{
+    background-color: white;
+    height: 2px;
+    width: 40px;
+}
+.dropDown ul{
+    list-style: none;
+}
+.dropDown ul{
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+    padding : 20px;
+    justify-content: space-between;
+    align-items: center;
+    color: white;
+    background-color: black;
+    height: 800px;
+    width: 300px;
+    border-radius: 0px 20px 20px 0px;
+}
+.dropDown li{
+    font-size: 120%;
```

Comment:
What if I view in 4k Screen, How big this would be ?

Reply:
It looks the same ... 
but anyway i have converted to pixels size